server/migrations/init.sql

```diff
@@ -11,24 +11,24 @@ CREATE EXTENSION citext;
 CREATE TABLE IF NOT EXISTS rentr_user (
     id SERIAL PRIMARY KEY,
     email CITEXT UNIQUE NOT NULL,
-    password VARCHAR(500) NOT NULL
+    password VARCHAR CHECK (length(password) <= 100) NOT NULL
 );
 
 INSERT INTO rentr_user(email, password) VALUES 
-('bosco@gmail.com', 'bosco123'),
-('justin@gmail.com', 'justin123'),
-('ronnie@gmail.com', 'ronnie123'),
-('azizul@gmail.com', 'azizul123'),
-('nathan@gmail.com', 'nathan123');
+('bosco@gmail.com', '$2b$10$b8/KshrtCRlab2qos7M1zO8cbAB7G4bB3RsN7bH8b21L9YiKbc51G'),
```

Comment:
Are these changes related?

Reply:
@NathanCarriere May not be related. But as we're storing the encrypted password having > 6 characters in the `/user/registration` POST API endpoint, this is just trying to ensure the data stored in the DB matches the constraints.